.gitignore

```diff
@@ -3,3 +3,5 @@
 .project
 /target/
 /work/
+.idea/
+extra-tool-installers.iml
```

Comment:
Please add a linefeed at the end of the file.
(it won't change the code behaviour, but it'll make github happier, make subsequent changes that add to the file clearer, and that'll make me happier).